src/echo-common/src/pino-logging-target.ts

```diff
@@ -0,0 +1,35 @@
+import pino, { Logger, LoggerOptions } from 'pino'
```

Comment:
@C3ntraX are you okay with this library?

Reply:
Yeah saw this. I don't know `pino`. It seems like a more lightweight version.
@julianheckmann Why did you choosed this lib and have you experience with other libs like winston?

Reply:
@C3ntraX pino is not only more lightweight but also much more faster. It was created by one of node.js core contributors. Here you can see some [benchmarks](https://github.com/pinojs/pino/blob/HEAD/docs/benchmarks.md). It is growing a lot in popularity, currently at 2m downloads. And since it covers all of the use-cases we currently need why not go with it :)

Reply:
Seems fine to me. We don't need colorization with morgan or different transport channels with or without docker etc. We need just production or not. It should be good enough and the benchmarks are good.